Ui/Component/Listing/Columns/CrossBorderClassActions.php

```diff
@@ -57,6 +65,16 @@ public function prepareDataSource(array $dataSource)
     {
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as &$item) {
+
+                /**
+                 * overriding cross border type id int with the label
+                 */
+                if(isset($item['cross_border_type_id'])) {
```

Comment:
There should be a constant for this string somewhere already. If there isn't, please create one

Reply:
brought in the CrossBorderClassInterface which had the constant.